Common/src/main/java/com/blamejared/crafttweaker/impl/command/type/MiscCommands.java

```diff
@@ -13,113 +12,133 @@
 import net.minecraft.network.chat.TranslatableComponent;
 import net.minecraft.server.level.ServerPlayer;
 
+import java.util.stream.Stream;
+
 public final class MiscCommands {
     
+    private static final String DOCS = "https://docs.blamejared.com";
+    private static final String DISCORD = "https://discord.blamejared.com";
+    private static final String ISSUES = "https://github.com/CraftTweaker/CraftTweaker/issues";
+    private static final String PATREON = "https://patreon.com/jaredlll08";
+    
     private MiscCommands() {}
     
-    public static void registerCommands() {
+    public static void registerCommands(final ICommandRegistrationHandler handler) {
         
-        CTCommands.registerCommand(new CommandImpl("give", new TranslatableComponent("crafttweaker.command.description.give"), builder -> {
-            builder.requires((source) -> source.hasPermission(2))
-                    .then(Commands.argument("item", IItemStackArgument.get())
-                            .executes(context -> {
-                                ExpandPlayer.give(context.getSource()
-                                        .getPlayerOrException(), context.getArgument("item", IItemStack.class), -1);
-                                return Command.SINGLE_SUCCESS;
-                            })
-                    );
-        }));
+        handler.registerRootCommand(
+                "give",
+                new TranslatableComponent("crafttweaker.command.description.give"),
+                builder -> builder.requires((source) -> source.hasPermission(2)) // TODO("Permission API")
```

Comment:
Not specifically, they were made OP only to fix anyone being able to use `/ct give`, but since we have an actual permission system now, we need to look and see *what* actually should be op only vs open to everyone